app/views/webhooks/edit.njk

```diff
@@ -14,9 +15,9 @@
 {% block mainContent %}
 <div class="govuk-grid-column-two-thirds">
   {{ govukBackLink({
-    text: "Back to Webhooks",
+    text: "Back to Webhook" if isEditing else "Back to Webhooks",
```

Comment:
```suggestion
    text: "Back to Webhook details" if isEditing else "Back to Webhooks",
```
Maybe makes more sense, as the user is kind of on the webhook already, if they are editing it? Not sure though!